docs/Framework.md

```diff
@@ -708,9 +708,9 @@ typedef NcUint32    NcOid; // Object id
 
 ### NcOrganizationId
 
-Unique 24-bit organization ID:
+Unique 24-bit organization id:
 
-- IEEE public Company ID (public CID) or
+- IEEE public Company id (public CID) or
```

Comment:
This is the problem with aiming for consistency when also referencing other specs... IEEE capitalise Company ID... https://standards.ieee.org/products-programs/regauth/cid/

Reply:
Yes, since we are referring to their term I think we should leave this one capitalised.